tools/grpc-swagger/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openjdk:8
+
+EXPOSE 8080
+
+RUN wget https://github.com/grpc-swagger/grpc-swagger/releases/latest/download/grpc-swagger.jar
```

Comment:
maybe we should pin this instead of `latest`?

Reply:
👍  good call